src/django/pfb_analysis/functions.py

```diff
@@ -0,0 +1,14 @@
+from django.db.models.expressions import Func
+
+
+class ObjectAtPath(Func):
```

Comment:
Yeah, I agree, but https://docs.djangoproject.com/en/1.10/ref/contrib/postgres/fields/#jsonfield is focused on filtering and I don't see anything that would fit the bill at https://docs.djangoproject.com/en/1.10/ref/models/expressions/

The double-underscore syntax has been extended to traversing JSONField values in some contexts, e.g.
```
AnalysisJob.objects.filter(overall_scores__overall_score__score_normalized__gt=1)
```
works, but
```
AnalysisJob.objects.annotate(overall_score=F('overall_scores__overall_score__score_normalized'))
```
says "FieldError: Cannot resolve keyword 'overall_score' into field. Join on 'overall_scores' not permitted."